packages/sync-actions/test/customer-sync.spec.js

```diff
@@ -533,4 +534,38 @@ describe('Actions', () => {
       },
     ])
   })
+
+  test('should build setAuthenticationMode sync action', () => {
+    let before = {
+      authenticationMode: 'Password',
+    }
+    let now = {
+      authenticationMode: 'ExternalAuth',
+    }
+
+    let actual = customerSync.buildActions(now, before)
+    let expected = [
+      {
+        action: 'setAuthenticationMode',
+        authenticationMode: now.authenticationMode,
+      },
+    ]
+    expect(actual).toEqual(expected)
+
+    before = {
+      authenticationMode: 'ExternalAuth',
+    }
+    now = {
+      authenticationMode: 'Password',
+    }
+
+    actual = customerSync.buildActions(now, before)
+    expected = [
+      {
+        action: 'setAuthenticationMode',
+        authenticationMode: now.authenticationMode,
+      },
```

Comment:
shouldnt the action also include `password` ref https://docs.commercetools.com/api/projects/customers#set-authenticationmode, wdyt?

Reply:
totally right. otherwise i got:

`{
    "statusCode": 400,
    "message": "'password' should not be empty.",
    "errors": [
        {
            "code": "InvalidOperation",
            "message": "'password' should not be empty.",
            "action": {
                "action": "setAuthenticationMode",
                "authMode": "Password"
            },
            "actionIndex": 1
        }
    ]
}`

thanks mate.